pkg/fanal/analyzer/analyzer.go

```diff
@@ -484,7 +484,8 @@ func (ag AnalyzerGroup) PostAnalyze(ctx context.Context, files *syncx.Map[Type,
 			Options: opts,
 		})
 		if err != nil {
-			return xerrors.Errorf("post analysis error: %w", err)
+			log.Logger.Debugf("Analysis error: %s", err)
+			return nil
```

Comment:
@DmitriyLewen Can you open another PR for this change?

Reply:
Done - #4037